noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -227,11 +226,11 @@ impl PrivateContext {
             // We didn't get a match meaning the cached result is stale. We fetch new values from oracle and instruct
             // protocol circuits to validate them by storing the validation request in context.
             let request = get_key_validation_request(pk_m_hash, key_index);
+            let request_and_generator = KeyValidationRequestAndGenerator { request, sk_app_generator: sk_generators[key_index] };
```

Comment:
Do you think this is an issue?

The issue essentially is that we don't enforce that for example apps don't use ovsk_m instead of nsk_m for nullification, right?

Reply:
Not an issue, just that it can be used, more a feature as you might be able to build interesting things that are using similar keys but different types of signatures, could be for stuff like "signed messages" if you want it to be specific to some app etc.